genestack_client/settings/genestack_config.py

```diff
@@ -183,6 +196,32 @@ def save(self):
                         password_element.appendChild(document.createTextNode(user.password))
                         user_element.appendChild(password_element)
 
+            elif user.token:
+                try:
```

Comment:
This whole block should be cut out to separate method (`store_securemost`, for example), to be used for token and password.